### PR TITLE
Athenian Civic Center

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**

### CODE REVIEW

**Comment by indoptogopt** [comment]
Could you look over the text again? I cant  understand some sentences or make out what you're trying to say

File: 0-ad-history-mod-package/simulation/templates/structures/athen/civil_centre.xml [hunk 1]
```diff
@@ -6,6 +6,11 @@
   <Identity>
     <Civ>athen</Civ>
     <SpecificName>Agora</SpecificName>
+    <History>
+    	Originally, Athens was ruled by a line of kings, descendants of Theseus. However, like many other Greek City. However, Athenian has undoubtedly become the most famous. The democracy had many iterations, as people like Solon and Cleisthenes reformed the laws.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/structures/athen/civil_centre.xml:10]:
```suggestion
    	Originally, Athens was ruled by a line of kings. But, like many other Greek City states, Athens became a republic. Athenians democracy had many iterations, with many people reforming its laws.

File: 0-ad-history-mod-package/simulation/templates/structures/athen/civil_centre.xml [hunk 2]
```diff
@@ -6,6 +6,11 @@
...
+    	The primary actors in the democracy were the Athenian citizens, men over the age of twenty. Citizens were appointed, elected, or chosen by lot to positions of magistrate, tribunal, or council and could attend the assembly which was the primary place of government.
+    	Democracy in ancient Athens was semi stable, every so often a tyrant would overthrow it, usually to the oppression of the populace. To try to protect themselves against it, there became the practice of ostracism. Yearly the Athenian people who decide whether an ostracism. If so, the citizens of Athens would write on shards of broken pottery the name of a politician who they thought was getting too much power. Whoever had the most votes was exiled for ten years.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/structures/athen/civil_centre.xml:12]:
```suggestion
    	Democracy in ancient Athens was semi stable, every so often a tyrant would overthrow it, usually to the oppression of the populace. To try to protect themselves against it, the Athenians began the practice of ostracism. Every year the Athenian people who decide whether to conduct an ostracism. If so, the citizens of Athens would write on shards of broken pottery the name of a politician who they thought was getting too much power. Whoever had the most votes was exiled for ten years.
```

**Comment by TheShadowOfHassen** [comment]
Is that better?

File: 0-ad-history-mod-package/simulation/templates/structures/athen/civil_centre.xml [hunk 3]
```diff
@@ -6,6 +6,11 @@
   <Identity>
     <Civ>athen</Civ>
     <SpecificName>Agora</SpecificName>
+    <History>
+    	Originally, Athens was ruled by a line of kings. But, like many other Greek City states, Athens became a republic. Athenians democracy had many iterations, with many people reforming its laws.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/structures/athen/civil_centre.xml:10]:
```suggestion
    	Originally, Athens was ruled by a line of kings, but, like many other Greek city states, eventually became a republic. Athens' democracy had many iterations, with many people reforming its laws such as Solon and Cleisthenes.
```

File: 0-ad-history-mod-package/simulation/templates/structures/athen/civil_centre.xml [hunk 4]
```diff
@@ -6,6 +6,11 @@
...
+    	The primary actors in the democracy were the Athenian citizens, men over the age of twenty. Citizens were appointed, elected, or chosen by lot to positions of magistrate, tribunal, or council and could attend the assembly which was the primary place of government.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/structures/athen/civil_centre.xml:11]:
```suggestion
    	The primary actors in the democracy were the Athenian citizens, men over the age of twenty. Citizens were appointed, elected, or chosen by lot to positions of magistrate, tribunal, or council and could attend the assembly, which was the primary place of government.
```

File: 0-ad-history-mod-package/simulation/templates/structures/athen/civil_centre.xml [hunk 5]
```diff
@@ -6,6 +6,11 @@
...
+    	Democracy in ancient Athens was semi stable, every so often a tyrant would overthrow it, usually to the oppression of the populace. To try to protect themselves against it, the Athenians began the practice of ostracism. Every year the Athenian people who decide whether to conduct an ostracism. If so, the citizens of Athens would write on shards of broken pottery the name of a politician who they thought was getting too much power. Whoever had the most votes was exiled for ten years.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/structures/athen/civil_centre.xml:12]:
```suggestion
    	Democracy in ancient Athens was semi-stable, every so often a tyrant would overthrow it, usually to the oppression of the populace. To try to protect themselves against it, the Athenians began the practice of ostracism. Every year the Athenian people decided whether to conduct an ostracism. If so, the citizens of Athens would write on shards of broken pottery the name of a politician who they thought was getting too much power. Whoever had the most votes was exiled for ten years.
```

**Review by indoptogopt — CHANGES_REQUESTED** [review]

**Comment by indoptogopt** [comment]
> Is that better?

Yup, thank you

**Comment by TheShadowOfHassen** [comment]
Thank _you_ for looking it over.

**Pull request merged**